### PR TITLE
Rebrand

### PR DESCRIPTION
**Pull request by JasonStoltz**
- Updated “Swiftype” references with “Elastic” in the readme and in the code.
- Renamed the published package to `elastic-enterprise-search`
- Changed versioning to be pre-release again, 0.1.0, since we are not yet in GA, just like we are doing for App Search.
- Updated copyright and license. Project is now Apache 2.0.
- Sending the analytics headers: X-Swiftype-Client and X-Swiftype-Client-Version
- Enabled CircleCI.

### CODE REVIEW

**Comment by yakhinvadim** [comment]
Starting review

File: doc/api/index.rst [hunk 1]
```diff
@@ -3,5 +3,4 @@ Swiftype Enterprise Client API
```
Comment (yakhinvadim) [inline doc/api/index.rst]: Is the first line in file can be changed here?

File: README.md [hunk 2]
```diff
@@ -38,15 +40,15 @@ Retrieve your access token and a content source key after creating your content
...
   authorization_token = 'authorization token'
-  client = SwiftypeEnterpriseClient(authorization_token)
+  client = Client(authorization_token)
   documents = [
     {
       'id': '1234',
-      'url': 'https://github.com/swiftype/swiftype-enterprise-python',
-      'title': 'Swiftype Enterprise Python Github',
+      'url': 'https://github.com/elastic/enterprise-search-python',
+      'title': 'Elastic App Search Official Python Client',
```
Comment (yakhinvadim) [inline README.md]: Enterprise*

File: doc/api/swiftype_enterprise_client.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
-:class:`~swiftype_enterprise.SwiftypeEnterpriseClient`
+:class:`~elastic_enterprise_search.SwiftypeEnterpriseClient`
```
Comment (yakhinvadim) [inline doc/api/swiftype_enterprise_client.rst]: We can't change this `SwiftypeEnterpriseClient`? Same for line 6.

File: doc/index.rst [hunk 4]
```diff
@@ -17,7 +17,7 @@ the docs are on `ReadTheDocs <https://swiftype_enterprise.readthedocs.io/>`_.
```
Comment (yakhinvadim) [inline doc/index.rst]: There are a lot of swiftype references above this line. Do we want to keep them?

**Review by yakhinvadim — COMMENTED** [review]
I noticed a couple of places where you left Swiftype references. Do we need to keep them?

File: doc/index.rst (hunk 4 above)
Comment (JasonStoltz) [reply doc/index.rst]: @yakhinvadim Whoops. I meant to circle back and address those doc files. Sorry about that, will tag you again when they're updated.

Comment (JasonStoltz) [reply doc/index.rst]: @yakhinvadim I ended up deleting these docs. I don't really know if they were generated or manually maintained. However, none of our other Python client libraries contain docs like these, and I'd like to not maintain them going forward.

**Comment by JasonStoltz** [comment]
@yakhinvadim Ready for a second look.

**Review by yakhinvadim — APPROVED** [review]
LGTM 👍 

(105% reviewed)
image

**Comment by JasonStoltz** [comment]
You've gone above and beyond :)

**Comment by JasonStoltz** [comment]
- Updated “Swiftype” references with “Elastic” in the readme and in the code.
- Renamed the published package to `elastic-enterprise-search`
- Changed versioning to be pre-release again, 0.1.0, since we are not yet in GA, just like we are doing for App Search.
- Updated copyright and license. Project is now Apache 2.0.
- Sending the analytics headers: X-Swiftype-Client and X-Swiftype-Client-Version
- Enabled CircleCI.

**Pull request merged**